handlers/svg-open-street-map/osm-svg.py

```diff
@@ -48,23 +58,61 @@ def handle():
     # Get and validate the request contents
     contents = request.get_json()
     try:
+        LOGGER.debug("Validating request")
         validator = jsonschema.Draft7Validator(
             request_schema, resolver=resolver
         )
         validator.validate(contents)
     except ValidationError as e:
-        logging.error(e)
-        return jsonify("None"), 204
+        LOGGER.error(e)
+        return jsonify("Invalid request received!"), 400
 
     # Check preprocessor data
+    if "preprocessors" not in contents:
```

Comment:
Why is this separate from the if statement directly below? This is a clear `else` state.